packages/web-components/src/components/rux-input/rux-input.scss

```diff
@@ -62,57 +64,59 @@
         position: relative;
         box-sizing: border-box;
         order: 2;
-        border: 1px solid var(--color-border-interactive-muted);
+        // use box shadow instead of border to create borders
+        box-shadow: var(--color-border-interactive-muted) 0 0 0 1px inset;
         border-radius: var(--radius-base);
         font-family: var(--font-body-1-font-family);
         font-size: var(--font-body-1-font-size);
         font-weight: var(--font-body-1-font-weight);
+        //control-body line heights not yet implemented
+        line-height: var(--line-height-sm);
```

Comment:
If you update `@astrouxds/design-tokens` to 2.0.0-beta.10, you should have access to `--font-control-body-1-line-height. 

nit: we should also update all the other references to `font-body-1` to `font-control-body-1`

Reply:
Thank you! I didn't understand where the new ones were. I'll keep that in mind. Fixed. Also updated the other font-bodys to font-control-bodys